iodict/__init__.py

```diff
@@ -24,6 +24,18 @@
 import typing
 import uuid
 
+if os.uname().sysname == "Darwin":
+    import xattr
+
+    getxattr, setxattr, listxattr = (
+        xattr.getxattr,
+        xattr.setxattr,
+        xattr.listxattr,
+    )
+
+else:
+    getxattr, setxattr, listxattr = (os.getxattr, os.setxattr, os.listxattr)
+
```

Comment:
Do we want to do it this way, or would we prefer:

```
if os.name().sysname == "Darwin":
    from xattr import getxattr
    from xattr import setxattr
    from xattr import listxattr

else:
    from os import getxattr
    from os import setxattr
    from os import listxattr
```

I kind of prefer this way, but we are already importing os which would be required for the if statement in the first place. So it feels a bit silly to import methods from the os module when we already have the entire thing imported to begin with.

Reply:
++ They're functionally equivalent. Also from X import Y is just a convenience tool; python reads the whole module no mater how you import it, so running the multiple `from` imports doesn't save us anything.